src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DocumentMapping/AbstractDocumentMappingService.cs

```diff
@@ -277,20 +277,20 @@ public bool TryMapToHostDocumentPosition(IRazorGeneratedDocument generatedDocume
         return false;
     }
 
-    public bool TryMapToGeneratedDocumentOrNextCSharpPosition(IRazorGeneratedDocument generatedDocument, int hostDocumentIndex, out LinePosition generatedPosition, out int generatedIndex)
-        => TryMapToGeneratedDocumentPositionInternal(generatedDocument, hostDocumentIndex, nextCSharpPositionOnFailure: true, out generatedPosition, out generatedIndex);
+    public bool TryMapToGeneratedDocumentOrNextCSharpPosition(RazorCSharpDocument csharpDocument, int hostDocumentIndex, out LinePosition generatedPosition, out int generatedIndex)
```

Comment:
I noticed this method name and realized that the `IDocumentMappingService` method names might need a rethink in light of `IRazorGeneratedDocument` going away.

Reply:
Actually, that reminds me, I meant to see if I can delete `IRazorGeneratedDocument` entirely. Thanks!